plugins/modules/cloud/rackspace/rax_files_objects.py

```diff
@@ -582,7 +507,10 @@ def main():
 
     module = AnsibleModule(
         argument_spec=argument_spec,
-        required_together=rax_required_together()
+        required_together=rax_required_together(),
+        mutually_exclusive=[
+            ('src', 'dest'),
```

Comment:
Right now they are only mutually exclusive for `method=delete` or `type=meta`. For `method=put` and `type=file`, both can be specified (for actual files and not directories)

Reply:
okie, rolled back